src/aurora/keyfile.cpp

```diff
@@ -202,4 +202,18 @@ Common::SeekableReadStream *KEYFile::getResource(uint32 index, bool UNUSED(tryNo
 	return iRes.dataFile->getResource(iRes.resIndex);
 }
 
+std::vector<Archive::Resource *> KEYFile::getResourceListForDataFile(Common::UString dataFile) {
```

Comment:
That should take a `const Common::UString &`.

And the return could be a `std::vector<const Archive::Resource *>`

The method itself is const, too.